io.github.zaps166.QMPlay2.Devel.json

```diff
@@ -0,0 +1,190 @@
+{
+  "app-id": "io.github.zaps166.QMPlay2",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.6",
+  "sdk": "org.kde.Sdk",
+  "add-extensions": {
+      "org.freedesktop.Platform.ffmpeg-full": {
+          "directory": "lib/ffmpeg",
+          "version": "23.08",
+          "add-ld-path": ".",
+          "autodelete": false
+      },
+      "org.freedesktop.Platform.GL.default": {
+          "directory": "lib/dri",
+          "version": "23.08",
+          "autodelete": false
+      }
+  },
+  "cleanup-commands": [
+      "mkdir -p /app/lib/ffmpeg",
+      "mkdir -p /app/lib/dri"
+  ],
+  "command": "QMPlay2",
+  "rename-icon": "QMPlay2",
+  "rename-desktop-file": "QMPlay2.desktop",
+  "rename-appdata-file": "QMPlay2.appdata.xml",
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--share=network",
+    "--socket=pulseaudio",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--filesystem=home",
+    "--filesystem=xdg-music:ro",
+    "--filesystem=xdg-videos:ro",
+    "--filesystem=xdg-run/pipewire-0:ro",
+    "--filesystem=/media",
+    "--filesystem=/mnt",
+    "--filesystem=/run/media",
+    "--system-talk-name=org.freedesktop.UDisks2"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "cleanup-commands": [
+      "mkdir -p /app/lib/ffmpeg",
+      "mkdir -p /app/lib/dri"
+  ],
+  "modules": [
+   {
+      "name": "libass",
+      "buildsystem": "autotools",
+      "config-opts": [
+        "--enable-harfbuzz",
+        "--enable-fontconfig"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/libass/libass/",
+          "tag": "0.17.0",
+          "commit": "bef4b43ef1882b77f789f611d9cd24271ccdd65b"
+        },
+        {
+          "type": "script",
+          "commands": [
+            "autoreconf -fiv"
+          ],
+          "dest-filename": "autogen.sh"
+        }
+      ]
+    },
+    {
+      "name": "libva",
```

Comment:
Remove it - we must not compile `libva`, because it'll find in default paths. Flatpak has already libva which finds drivers correctly.